devops/actions/cleanup/cleanup.js

```diff
@@ -1,3 +1,4 @@
 const path = process.env.GITHUB_WORKSPACE + '/*';
 console.log('Cleaning ' + path)
 require('child_process').execSync('rm -rf ' + path);
+require('child_process').execSync('rm -rf ' + process.env.GITHUB_WORKSPACE + '/.git');
```

Comment:
Don't we want to remove all hidden files/directories and not just `.git`?

Reply:
using `const path = process.env.GITHUB_WORKSPACE + '/{*,.*}';` at line 1 should work.

Reply:
I will prepare a PR with this tomorrow - want to give https://github.com/intel/llvm/pull/10074 more time to expose any potential issues post-commit on its own (although I hope that none would appear, of course).

Reply:
https://github.com/intel/llvm/pull/10095